src/coreComponents/dataRepository/Group.hpp

```diff
@@ -1471,7 +1471,7 @@ class Group
    * @param[in] group A group that is passed in to the initialization functions
    *                  in order to facilitate the initialization.
    */
-  virtual void InitializePostInitialConditions_PostSubGroups( Group * const group )
+  virtual void initializePostInitialConditionsPostSubGroups( Group * const group )
   {
```

Comment:
This seem to be the longest function name, can we give it another more appropriate name?

Reply:
Yes this is true, but I could only find this name like 4 times in the whole codebase.
Also the wording is quite comprehensive and every word has its importance.
Se could transform it into `initPostInitialCondsPostSubGrps`, but is it really worth it for so unfrequent use?